### PR TITLE
feat: Add support for Polish language

### PR DESCRIPTION
**Pull request by kkrawczykpl**
Polish translation for issue #43.

I added Polish language files and possibility to select Polish language in settings.

### CODE REVIEW

**Comment by julienc91** [comment]
Hey @kkrawczykpl, thanks for this great PR! I have zero knowledge about Polish, so I have to trust you on the content :smile: If I were to add or modify any of the translated content in the future, would you mind if I ping you to update the Polish version accordingly? Don't worry I don't plan a lot of upcoming changes, so don't expect more than 1h twice a year :sweat_smile: It's just that I hate websites that are using google translate or similar, and I don't want to add a language if there's no one to maintain it :slightly_smiling_face:

**Comment by kkrawczykpl** [comment]
@julienc91 Sure! Just ping me, I can translate any content 😁 If you have any code-related issues (Preferably in JS/TS) then I can take care of it too :)

**Comment by julienc91** [comment]
@kkrawczykpl ok great! I have to adapt [my other repo](https://github.com/julienc91/skin_exporter) to export skin descriptions in Polish before merging this PR, I'm not sure I'll have the time to do it this week-end, but I'll do my best :slightly_smiling_face: 

Regarding other frontend tasks in this repo, here are a few ideas I have in mind:

* new homepage: I've been working on a refactoring for a few weeks now, but I have nothing really conclusive... I wanted the homepage to better explain what the site does, put more emphasis on the link to the csgo listing page. This is quite a complex task since I'm not sure what I want :sweat_smile:  
* migrate to TS: that's a tedious task, I never really had the courage to start...
* find how to integrate the "pro-players" section: if you don't know what it is, this is because there is no link to it anywhere (I implemented it but had backend difficulties I managed to fix in the meantime, but never took the time to think about how I could integrate it). If you want to have a look, it's at `/en/counter-strike-global-offensive/teams`

These are the ideas I've had in mind for a long time, but if you want to contribute in any other way, I'm all ears :slightly_smiling_face:

File: frontend/locales/pl/common.json [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+      "home": "Strona główna"
+    },
+    "changelog": {
+      "title": "Co nowego na LionSkins:",
+      "2019_10_20_001": "Dodano do oferty Klasyczny Nóz",
+      "2019_10_21_001": "Od teraz możesz sprawdzić ceny skinów do rękawic!",
+      "2019_11_19_001": "Dodano nowe noże z kolekcji Shattered Web",
+      "2020_05_08_001": "Dodano Noże Vanilla",
+      "2020_05_09_001": "Od teraz możesz sprawdzać ceny przedmiotów z serwisu CS.Money!",
+      "2020_05_22_001": "Zobacz, jakie skórki posiadasz i określ cenę twojego ekwipunku Steam.!",
```
Comment (julienc91) [inline frontend/locales/pl/common.json:17]: Is there a typo in the punctuation?

File: frontend/locales/pl/faq.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        "title": "Dlaczego tylko Counter-Strike?",
+        "content": "Na początku chcieliśmy dodać więcej gier, ale nie mieliśmy na to ani czasu, ani środków... Ale pamiętamy o tym, może kiedyś to nastąpi!"
+      },
+      "part2": {
+        "title": "Dlaczego nie sugerujecie danego rynku?",
+        "content": "Na pewno strona którą sugerujesz jest wspaniała, ale nie możemy uwzględnić wszystkich możliwych rynków. Jeżeli jednak będzie liczny odzew a sprawdzimy że produkt który oferujesz jest na pewno bezpieczny być może go dodamy w przyszłości."
+      },
+      "part3": {
+        "title": "Podana cena nie była właściwa!",
+        "content": "Mimo że dążymy do aktualizacji cen w czasie rzeczywistym, nie jesteśmy akutalnie w stanie tego zrobić ze względów technicznych."
```
Comment (julienc91) [inline frontend/locales/pl/faq.json:17]: Part of the translation seems to have been missed

**Comment by julienc91** [comment]
@kkrawczykpl I finished adapting the backend to welcome this new language. Can you just have a quick look at the two remarks I left? Thanks!

File: frontend/locales/pl/common.json (hunk 1 above)
Comment (kkrawczykpl) [reply frontend/locales/pl/common.json]: Done

File: frontend/locales/pl/faq.json (hunk 2 above)
Comment (kkrawczykpl) [reply frontend/locales/pl/faq.json]: Yeah, I shortened this sentence strongly. It had the same meaning, but I added the missing part.

**Pull request merged**